packages/ses/scripts/hermes.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
```

Comment:
Might need a comment and a more specific name, like `hermes-test.sh`. I’m fine with a test script not being portable to Windows, but not a build script. (This is just a test script, aye?)